docs/gallery/interactive/pyfar_audio_objects.ipynb

```diff
@@ -0,0 +1,413 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets break down the information one by one. \n",
+    "\n",
+    "**Domain:** The signal is in the time domain, because we last accessed its time data above. We can get and set this property with the `signal.domain` attribute but it is of not much interested when working with Signal objects because they are automatically converted between the time and frequency domain, when it is required."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.domain"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Signal Type:** It is an *energy* Signal, which is important to note and understand. Energy signals are finite signals with finite energy. Examples for these kinds of signals are impulse responses. *Power* signals on the other hand have an infinite duration and energy. Examples are noise or sine signals of which we typically observe a block of finite size. The signal type is a read only property that is set by the DFT normalization introduced below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.signal_type"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**DFT Normlaization:** For *power* Signals, different DFT normalization can be used that differently scale the values stored in `signal.freq`. For *energy* signals, the DFT normalization is always 'none'. For more information refer the `Fast Fourier Transform` examples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal.fft_norm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Signal cshape, length, and caxis\n",
+    "\n",
+    "Signals can contain multiple channels of audio data and the shape of the data inside the audio object is often important. There multiple attributes describing this.\n",
+    "\n",
+    "**Channel shape:** The *channel shape*, in short `cshape` gives the shape of the data inside an audio object but ignores the number of samples or frequency bins. For example our first Signal created above has one channel and 4 samples, hence the data is of chape `(1, 4)`. Because the `cshape` ignores the samples it is `(1, )`. There are two ideas behind this. First, digital signal processing methods often work on channels, and second the length of the signal might be different in the time and frequency domain."
```

Comment:
```suggestion
    "**Channel shape:** The *channel shape*, in short `cshape` gives the shape of the data inside an audio object but ignores the number of samples or frequency bins. For example our first Signal created above has one channel and 4 samples, hence the data is of shape `(1, 4)`. Because the `cshape` ignores the samples it is `(1, )`. There are two ideas behind this. First, digital signal processing methods often work on channels, and second the length of the signal might be different in the time and frequency domain."
```
Shouldn't this be `shape`?